attribution-graphs-frontend/feature_examples/init-feature-examples-list.js

```
@@ -0,0 +1,116 @@
+window.initFeatureExamplesList = function({renderAll, visState, sel}){
+  var sel = sel.select('.feature-example-list')
```

Biome (1.9.4)

[error] Shouldn't redeclare 'sel'. Consider to delete it or rename it.

'sel' is defined here:


(lint/suspicious/noRedeclare)